src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx

```diff
@@ -1374,6 +1374,138 @@ For more information, see [Python agent attributes](/docs/agents/python-agent/at
   </Collapser>
 </CollapserGroup>
 
+## AI monitoring [#ai-monitoring]
+
+This section includes Python agent configurations for setting up AI monitoring. 
+
+<Callout variant="important">
+You need to set [distributed_tracing](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#dt-main) to `true` to capture trace and feedback data. It is turned on by default in Python agent versions 7.0.0.166 and higher. 
```

Comment:
Underscore in distributed tracing seems to be adding a double space.

Also, it's instead of It is